tests/e2e/tests/pilot/testdata/config.yaml.tmpl

```diff
@@ -11,12 +11,12 @@ data:
       drainDuration:          2s
       parentShutdownDuration: 3s
 {{if eq .Auth.String "NONE"}}
-      discoveryAddress:       istio-pilot.{{.IstioNamespace}}:15007
+      discoveryAddress:       istio-pilot.{{.IstioNamespace}}:8080
```

Comment:
Why 8080? We don't want observability anymore for pilot? This is not a build and test improvement, it changes how the system behaves.

Reply:
The 15007 feature was half done and not working with larger number of pods. Testing with what we'll use.